src/main/java/com/github/dockerjava/core/async/ResultCallbackTemplate.java

```diff
@@ -87,4 +95,13 @@ public RC_T awaitCompletion(long timeout, TimeUnit timeUnit) throws InterruptedE
         completed.await(timeout, timeUnit);
         return (RC_T) this;
     }
+
+    protected RuntimeException getFirstError() {
```

Comment:
I usually mark `@CheckForNull` and `@Nonnull` to know what logic about nullness i implemented in method.